examples/riot/hello_lf/Makefile

```diff
@@ -8,7 +8,7 @@ LF_MAIN ?= HelloLF
 # CFLAGS += -DNETWORK_CHANNEL_COAP_RIOT
 
 # Execute the LF compiler if build target is "all"
-ifeq ($(MAKECMDGOALS),all)
+ifeq ($(firstword $(MAKECMDGOALS)),all)
```

Comment:
Out of curiosity, why is it necessary for `all` to be the first target?

Reply:
Because they are executed in order and we first need to build the project and then run term.
But actually the fix is not so much about that. The main issue was, that before it only worked if the argument is `all`. So if anything came after `all` it would not work.

This way it is more relaxed and only checks if `all` comes first and doesn't care what comes after ;)